scripts/support/create_ge_images/create_secgroup

```diff
@@ -23,13 +23,15 @@
 #
 
 
-nova secgroup-create allopen
+export PATH=/usr/bin:$PATH ; export PYTHONPATH=/usr/lib/python2.7/dist-packages
```

Comment:
Why do you need export these PATHs? What happen if python2.7 is not installed in that location?